packages/cli/src/config/extensions/github.ts

```diff
@@ -52,7 +64,15 @@ export async function cloneFromGit(
         // We let git handle the source as is.
       }
     }
-    await git.clone(sourceUrl, './', ['--depth', '1']);
+    await simpleGit()
+      .env(cloneEnv)
+      .clone(sourceUrl, destination, ['--depth', '1']);
```

Comment:
Afaict, no hooks run on `clone` ever (there is no post-clone hook for git that I can find).

Can we remove everything except the part below where we create the hooks disabled dir prior to  doing any checkouts?